### PR TITLE
Adding two heuristics for networking

### PR DESCRIPTION
**Pull request by cccs-kevin**
Partially addresses https://cccs.atlassian.net/browse/AL-2762

### CODE REVIEW

File: vipermonkey_.py [hunk 1]
```diff
@@ -210,6 +211,25 @@ def execute(self, request: ServiceRequest) -> None:
                             urldownload_ioc_sec.set_heuristic(6)
                             sub_action_section.add_subsection(urldownload_ioc_sec)
 
+                    elif action.lower() == "CreateObject".lower() and "WinHTTPRequest".lower() in param.lower():
+                        winhttpreq_heur = Heuristic(7)
+                        _ = ResultSection(
```
Comment (cccs-jh) [inline vipermonkey_.py:216]: no need for the assignment

File: vipermonkey_.py [hunk 2]
```diff
@@ -210,6 +211,25 @@ def execute(self, request: ServiceRequest) -> None:
...
+
+                    elif action.lower() == "POST".lower():
+                        post_heur = Heuristic(8)
+                        post_sec = ResultSection(
+                            post_heur.name,
+                            post_heur.description,
+                            heuristic=post_heur,
+                            parent=sub_action_section,
+                        )
+                        _ = add_tag(post_sec, "network.static.uri", param)
```
Comment (cccs-jh) [inline vipermonkey_.py:231]: same here

**Review by cccs-jh — APPROVED** [review]

File: vipermonkey_.py (hunk 1 above)
Comment (cccs-kevin) [reply vipermonkey_.py]: resolved

File: vipermonkey_.py (hunk 2 above)
Comment (cccs-kevin) [reply vipermonkey_.py]: resolved

**Review by cccs-jh — APPROVED** [review]

**Pull request merged**